src/test/java/com/dsrm/dsrmbackend/UserControllerIntegrationTests.java

```diff
@@ -87,4 +109,33 @@ void retrieveUsersWithoutRoles() throws Exception {
 
     }
 
+
+    @Test
+    void validPatchExistingUser() throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
+        UserRolesOnlyDTO userRolesOnlyDTO  = new UserRolesOnlyDTO();
+        List<Long> longs = new ArrayList<>();
+        longs.add(1L);
+        userRolesOnlyDTO.setRoles(longs);
+        this.mockMvc.perform(patch("/users/1/roles").contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(userRolesOnlyDTO))
+                ).andExpect(status().isOk());
+        Optional<User> optionalUser = userRepo.findById(1L);
+        Assertions.assertNotNull(optionalUser);
```

Comment:
This assertion won't work. we want to check if content of optional is null not optional itself. It should be
```Assertions.assertTre(optionalUser.isPresent())```